__tests__/app.test.js

```diff
@@ -56,6 +56,43 @@ describe("GET /api", () => {
   });
 });
 
+describe("GET /api/articles/articles:id", () => {
+  it("200: responds with individual article", () => {
+    return request(app)
+      .get("/api/articles/2")
```

Comment:
You're specifically looking for article 2, you should not be expecting any response on the keys, you should be expecting the specific data for that article.